src/inc/redirect/Mlp_Language_Negotiation.php

```diff
@@ -58,8 +58,20 @@ public function __construct(
 	 * @return array
 	 */
 	public function get_redirect_match( array $args = array() ) {
+		/**
+		 * Filters the allowed status for posts to be included in hreflang links.
+		 *
+		 * @since 2.8.0
+		 *
+		 * @param string[] $post_status Allowed post status.
+		 */
+		$post_status = (array) apply_filters( 'multilingualpress.hreflang_post_status', array(
```

Comment:
Can you please change the filter name to `multilingualpress.redirect_post_status`, and also adapt the description (PHPDoc)? Thanks.